.circleci/config.yml

```diff
@@ -598,22 +719,152 @@ jobs:
           name: invalidate cloudfront
           command: |
             python3 CircleciScripts/cloudfront_invalidate.py sdk_s3_release  "${S3_RELEASE_DISTRIBUTION_ID}" "latest/$sdkName.zip"
+ 
+  bump_sdk_version:
+    docker:
+      - image: circleci/android:api-27-alpha
```

Comment:
any specific reason to use android image for  this operation?

Reply:
we don't need to build code in this job.  as xcode does not have a free plan. I use android image so that we can test it in our personal github reposition. another reason is that the code is also used in android project. using same image make it easier to main our code